stablehlo/reference/Ops.cpp

```diff
@@ -1074,6 +1093,28 @@ Tensor evalClzOp(const Tensor &operand, ShapedType resultType) {
   return result;
 }
 
+Tensor evalCollectiveBroadcastOp(
+    const Tensor &operand, SmallVector<SmallVector<uint32_t>> replicaGroups,
+    ChannelId channelId, Process *process) {
+  if (!process)
+    llvm::report_fatal_error(
+        "collective_broadcast is only supported when run via "
+        "interpreter.run_parallel");
+
+  ProcessGroups processGroups;
+  if (channelId <= 0) processGroups = process->crossReplica(replicaGroups);
+  if (channelId > 0) processGroups = process->crossPartition(replicaGroups);
+
+  for (auto processGroup : processGroups)
+    if (llvm::find(processGroup, process->getId()) != processGroup.end())
+      return process->rendezvous(processGroup, channelId, operand)
+          .lookup(processGroup[0]);
```

Comment:
Would `processGroups.findGroup` work here? Perhaps you can update the implementation of that to use `llvm::find`, perhaps in a separate change.

Reply:
Done.